evolvedbinary/prosemirror-lwdita
# Redirect the user to petalBot install Page

When generating the token the backend will check if petalBot is installed or not, then we can redirect the user to installation page.

How to test this PR:
1. Checkout the branch
2. Enable cors
3. Start the demo `yarn start:demo`
4. Start the backend `yarn start:backend`
5. Make sure you have a clean state, uninstall petal-demo and remove it from authorized list.
6. Visit `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&branch=main&referer=htwsrhtshrts`
7. Make sure you are redirected to GitHub OAuth page.
8. Make sure you are redirected to GitHub petal demo install page.
9 Visit `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&branch=main&referer=htwsrhtshrts` again.
10. Make sure you are redirected for a split second, and petal is always visible
